README.md

```diff
@@ -337,7 +338,7 @@ Install Velero, including all prerequisites, into the cluster and start the depl
 ```bash
 velero install \
     --provider azure \
-    --plugins velero/velero-plugin-for-microsoft-azure:v1.1.0 \
+    --plugins velero/velero-plugin-for-microsoft-azure:vv1.2.0 \
```

Comment:
The tag here and below has an extra `v`.

Reply:
How hard can one facepalm